### ISSUE TITLE
Documentation: Document process to create Release Acceptance Testing Discussions

### ISSUE THREAD

**Issue by jprestop** [dtcenter/METplus#2156]
## Describe the Task ##
Document the process to create a Release Acceptance Testing Discussion for each development, bugfix, and official coordinated release in METplus Discussions. During the NOAA METplus User Telecon on July 24, 2023, we discussed wether we should have a separate Release Acceptance Testing Discussion for each development release (i.e. separate discussions for beta1/2/3 and rc1/2), or use a single one for each coordinated release (i.e. combine beta1/2/3 and rc1/2 in the same discussion)?

Pushed this issue forward to the coordinated METplus 6.0.0 release after we discuss the details during the coordinated METplus 5.1.0 release retrospective meeting.

### Time Estimate ###
<1 day

### Sub-Issues ###
Consider breaking the task down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
No hard deadline

### Funding Source ###
Ask Tara before starting

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED CYCLE ASSIGNMENT** label
- [x] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Task Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issues
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

**Comment by jprestop** [comment]
@georgemccabe brought it to my attention that there is no assignee for this issue.  Since @JohnHalleyGotway has created the Release Acceptance Testing Discussions in the past, I have assigned this issue to @JohnHalleyGotway.  We do have a topic of discussion regarding the Release Acceptance Testing Discussions to bring up either at the METplus User's Telecon or at the EMC/NOAA Release Retrospective which is: If rc1 Release Acceptance Testing will contain all of the beta release information, should we then only have one Release Acceptance Testing going forward?  That may or may not have an impact on how this documentation is written.
